frontend/src/pages/auth/components/auth-layout/styles.module.scss

```diff
@@ -71,9 +71,7 @@
 }
 
 .heroTitle {
+  width: 615px;
```

Comment:
Let’s better specify a `max-width: 615px` and `width: 100%` to make it responsive